release-engineering/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
+  - sig-release-leads
```

Comment:
Personal opinion: I don't think SIG Chairs should implicitly be subproject owners.  You already have a chair here.

I would like to see someone from Google here, either @calebamiles or someone else who's been helping cut .deb's and .rpm's, from the perspective of our interest in debottlenecking.

Reply:
Do you mean "explicitly"?

I think sig-release-leads is redundant across these files' approvers.  There's still sig-release-leads at the next higher node's OWNERS file right?

Let's explicitly fully delegate unless there's an instance specific reason to not, but my initial feeling is that would be a sign of a prematurely formed and understaff subproject.